### ISSUE TITLE
Admin Order with Send order emails disabled still sends emails, but removes box to uncheck

### ISSUE THREAD

**Issue by IanFiretoys** [magento/magento2#30188]
### Preconditions (*)

1. 2.3.5-p2
2. stores > configuration > sales > sales emails > order > enabled = No

### Steps to reproduce (*)

1. disable order confirmation emails
2. in admin, place a new order, new customer or existing
3. add items and checkout information
4. in the submit order block "Email Order Confirmation" box is no longer there to uncheck

### Expected result (*)

1. Order is placed with no confirmation email sent out as per global settings
2.

### Actual result (*)

1. Order is placed, but confirmation email is sent out dispite global settings
2.

The problem seems to lie with lines 26-32 of vendor/magento/module-sales/view/adminhtml/templates/order/create/totals.phtml

```
    <?php if ($block->canSendNewOrderConfirmationEmail()) : ?>
    <div class="admin__field admin__field-option field-email-order-confirmation">
        <input type="checkbox" id="send_confirmation" name="order[send_confirmation]" value="1" checked="checked"
               class="admin__control-checkbox"/>
        <label for="send_confirmation" class="admin__field-label"><?= $block->escapeHtml(__('Email Order Confirmation')) ?></label>
    </div>
    <?php endif; ?>

```
the php if function on lines 26/32 only seems to govern whether the box is **shown** rather than whether it is checked. once the box is hidden, it remains checked as marked on line 28, forcing the order email to be sent, even though it is disabled globally

the workaround I found is to extend this file and remove the if function to allow the user in admin to make the choice to send/not send with the checkbox, but a potentially better solution would to make the if function an elseif that would mark the box as unchecked if the `canSendNewOrderConfirmationEmail` returns negative as well as hide the block.

---
Please provide [Severity](https://devdocs.magento.com/guides/v2.3/contributor-guide/contributing.html#backlog) assessment for the Issue as Reporter. This information will help during Confirmation and Issue triage processes.

- [ ] Severity: **S0** _- Affects critical data or functionality and leaves users without workaround._
- [ ] Severity: **S1** _- Affects critical data or functionality and forces users to employ a workaround._
- [x] Severity: **S2** _- Affects non-critical data or functionality and forces users to employ a workaround._
- [ ] Severity: **S3** _- Affects non-critical data or functionality and does not force users to employ a workaround._
- [ ] Severity: **S4** _- Affects aesthetics, professional look and feel, “quality” or “usability”._

**Comment by IanFiretoys** [comment]
@magento give me 2.4-develop instance

**Comment by IanFiretoys** [comment]
I can confirm that the boxes are gone, but without a working email service I can't replicate the bug easily on a develop instance.

**Comment by Pavel-Fekete** [comment]
Not reproducible on the latest 2.4-develop

**Steps to reproduce:**

1. Disable order confirmation emails
![Disable](https://user-images.githubusercontent.com/67371013/98798832-fac49600-2416-11eb-9f9a-1821e3f370a9.png)
2. Go to Admin and create new order
![Create order](https://user-images.githubusercontent.com/67371013/98799132-63137780-2417-11eb-9d0f-6eecc50fd08c.png)
3. In the submit order block **"Email Order Confirmation"** box is no longer there to uncheck
![checkbox](https://user-images.githubusercontent.com/67371013/98799401-bab1e300-2417-11eb-9290-ab94898a16c7.png)
4. Submit the order

**Actual Result:**
Order is placed with no confirmation email sent out as per global settings

**Comment by engcom-Bravo** [comment]
Hello @IanFiretoys 

Thank you for your report.

This issue is already fixed in the latest Magento 2.4-develop.

So we are closing this issue. Please feel free to comment, reopen or create new ticket according to the [Issue reporting guidelines](https://github.com/magento/magento2/wiki/Issue-reporting-guidelines)

Thank you
